src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1184,14 +1184,33 @@ Prevent Long URL From Breaking Out of Container
     margin-right: 5px;
 }
 
-/*
-Glyphicon for success status messages
-*/
+/*//////////////////////////////////////////////////////////
+////////////////////// Status Icons //////////////////////
+//////////////////////////////////////////////////////////*/
 
-.icon-success:before {
+.icon-success:before,
+.icon-warning:before,
+.icon-danger:before,
+.icon-info:before {
+    /*noinspection CssNoGenericFontName*/
```

Comment:
Yea I agree there are some issues surrounding adding `/*noinspection CssNoGenericFontName*/` (I will point them out in the new issue accordingly).

Not IntelliJ specific however I do think there should be a comment like
`/* icon font, generic fallback not relevant */`. However, it comes off as strange because without a lint warning there is no context unless the developer knows why (we can't assume that), yet not including it means that we don't care about generic fallbacks at all (which I think is a bigger mistake).

Personally, then I think the best compromise I can think of is to include the IntelliJ specific inspection suppression.

If there is a better compromise that does not involve the IntelliJ specific inspection, I would definitely prefer it.